### ISSUE TITLE
Support for CuDNN 8.0.5?

### ISSUE THREAD

**Issue by apolinario** [cloudhan/jax-windows-builder#6]
Unless I missed something, all cuda111 releases here https://whls.blob.core.windows.net/unstable/index.html are compiled with cuDNN 8.2.x. Is there a plan to support cuDNN 8.0.5 for the pre-made releases?

And while that is not available, is there any way to get Jax working with cuDNN 8.0.5 on Windows?

**Comment by cloudhan** [comment]
No, there is no plan to add support for older version of cudnn. Since it will double the storage requirement. If you want the support, build yourself locally is the only way.

Asides from that, you can just download cudnn 8.2, extract it to any directory, then add the directory which contains the DLLs to your PATH environment varibable. IIRC, then you are good to go, it won't interfere with you existing environment.

**Comment by apolinario** [comment]
Got it, thank you! I believe older (mainly 8.0.5, so it matches the Linux Jax distribution, but 8.1 would work too) support could be cool for machine learning applications together with Jax, as TensorFlow current distributions do not support 8.2.x. But I understand storage reasons. 

Unfortunately just straight forward trying to build Jax with `python build/build.py --enable_cuda --cuda_version 11.1 --cudnn_version 8.0.5` didn't work out of the box either (will try to debug later)
